### PR TITLE
[Expression_Document]Add doc for built-in functions

### PR DESCRIPTION
**Pull request by cosmicshuai**
Fixes #

## Proposed Changes

Doc for built-in functions includes:
  - description
  - parameter signatures
  - examples

### CODE REVIEW

**Comment by Danieladu** [comment]
@vishwacsena this is shuai's PR for the update of R2 function. May you go througth it ?

File: experimental/common-expression-language/prebuilt-functions.md [hunk 1]
```diff
@@ -528,1363 +820,2361 @@ createArray('h', 'e', 'l', 'l', 'o')
...
 
-<a name="empty"></a>
 
-### empty
+<a name="dateReadBack"></a>
 
-Check whether a collection is empty.
-Return true when the collection is empty,
-or return false when not empty.
+### dateReadBcak
```
Comment (Danieladu) [inline experimental/common-expression-language/prebuilt-functions.md]: spell error

**Comment by Danieladu** [comment]
missing 'average' function

File: experimental/common-expression-language/prebuilt-functions.md [hunk 2]
```diff
@@ -239,20 +286,46 @@ addMinutes('<timestamp>', <minutes>, '<format>'?)
...
+
+```
+addProperty('<object>', '<property>', value)
+```
+
+| Parameter | Required | Type | Description |
+| --------- | -------- | ---- | ----------- |
+| <*object*> | Yes | Object | The JSON Object where you want to add a property |
+|<*property*>| Yes | String | The name of the property to add |
+|<*value*>| Yes | any | The value of the property |
```
Comment (Danieladu) [inline experimental/common-expression-language/prebuilt-functions.md]: return object, refer to the new json object

File: experimental/common-expression-language/prebuilt-functions.md [hunk 3]
```diff
@@ -528,1363 +820,2361 @@ createArray('h', 'e', 'l', 'l', 'o')
...
-<a name="not"></a>
+<a name="subtractFromTime"></a>
 
-### not
+### subtractFromTime
 
-Check whether an expression is false.
-Return true when the expression is false,
-or return false when true.
+Subtract a n
```
Comment (Danieladu) [inline experimental/common-expression-language/prebuilt-functions.md]: missing ### ticks

File: experimental/language-generation/README.md [hunk 4]
```diff
@@ -95,6 +95,15 @@ Load the template manager with your .lg file(s)
...
+    TemplateEngine lgEngine = new TemplateEngine().AddFiles(pathToLGFiles); 
+```
+
+For NodeJS
+```
+    let lgEngine = new TemplateEngine().addFiles(pathToLGFiles);
+=======
+
 ```c#
     TemplateEngine lgEngine = TemplateEngine.FromFiles(pathToLGFile);
```
Comment (vishwacsena) [inline experimental/language-generation/README.md]: These should be removed - right? FromFiles is no longer valid for both C# and JS

**Pull request merged**